draft-regext-rdap-x-media-type.md

```diff
@@ -244,12 +248,12 @@ consult [@!RFC9110] regarding caching and other uses of the `vary` header.
 
 [@?RFC6838, section 4.3] strongly discourages the creation of new parameters on existing
 media types to enable new features. As RDAP has always had extensions, it could be argued
-that adding an "extensions" parameter to the existing `application/rdap+json` media type
+that adding an "extensions" parameter to the existing 'application/rdap+json' media type
```

Comment:
Same comment as above. Maybe we just double quote it. I don't know.

Reply:
the let us use double quotes.

Reply:
OK, let me do so.